app/controllers/staff/events_controller.rb

```diff
@@ -7,10 +7,14 @@ def edit
 
   def show
     teammates = @event.teammates.accepted
+    event_stats = EventStats.new(@event)
```

Comment:
Actually, you can just do @event.stats to get an instance. Useful around the rest of the app is current_event.stats.